src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -311,7 +311,7 @@ def validate(self, params: dict) -> None:
 
     def _set_output_types(self) -> None:
         for output in self._outputs_map.values():
-            return_types = self._get_method_return_type(output.method)
+            return_types = self._get_method_return_type(output.method or "")
```

Comment:
`output.method` can be None. 
Is it OK or should an exception be raised ?

Reply:
I think an exception is a good idea.

Reply:
done.PTAL.